static/templates/gear_menu.hbs

```diff
@@ -119,8 +119,13 @@
                 </a>
             </li>
             {{/if}}
+            <li role="presentation" class="hide only-visible-for-spectators">
```

Comment:
Would it make sense to have base CSS for `only-visible-for-spectators` include `display: none` rather than have the `hide` element as well?